crates/uv/tests/it/edit.rs

```diff
@@ -4826,6 +4826,112 @@ fn add_group() -> Result<()> {
 
     let pyproject_toml = context.read("pyproject.toml");
 
+    insta::with_settings!({
```

Comment:
Can we add a few test cases where there are comments between the groups, to understand how they move around?

Reply:
👍 this is an unfortunately frequent cause of bug reports